scripts/new_version/new_version.sh

```diff
@@ -119,3 +119,7 @@ then
 
   echo "tests/cache/worlds/backwards_compatibility.wbt should not be modified by this script"
 fi
+# metainfo
+echo "Update metainfo file"
+$CURRENT_DIR/new_version_file.sh $old_version $new_version $WEBOTS_HOME/scripts/packaging/webots.metainfo.xml
+$CURRENT_DIR/new_version_file.sh "[0-9]\{4\}-[0-9]\{2\}-[0-9]\{2\}" "$(date +%F)" $WEBOTS_HOME/scripts/packaging/webots.metainfo.xml
```

Comment:
This will wipe the previous version every time a new version comes out. Idk if that's what we want. (See other note about the possibility of manual releases.)

Reply:
I agree. If we go the automated-update route, I would do something like that rather than relying on `new_version_file`. (One instance should prob be used to update the copyright date tho.) However, I'd hold off until we decide whether the release process will be automatic or manual or automatic. (See [comment](https://github.com/cyberbotics/webots/pull/6885#discussion_r2297106644)) If the former, I would omit this and just let the releasers update it. If the latter, we could probably auto-generate a changelog link, but the actual changelog in this file would be blank. @omichel thoughts?